css/css-sizing/contain-intrinsic-size/contain-intrinsic-size-009.html

```diff
@@ -52,12 +54,11 @@
 <script src="/resources/testharnessreport.js"></script>
 <script src="/resources/check-layout-th.js"></script>
 <script>
-const reference = document.getElementById("reference");
-const rowHeight = reference.clientHeight;
-const normalWidth = reference.clientWidth;
+const normalWidth = document.getElementById("reference-size-1").clientWidth;
 for (let test of document.querySelectorAll(".test.cis-width")) {
-  const numRows = test.size || 4;
-  test.dataset.expectedClientHeight = rowHeight * numRows;
+  let idText = "reference-size-";
+  idText += test.size ? test.size : 4;
+  test.dataset.expectedClientHeight = document.getElementById(idText).clientHeight;
```

Comment:
Sorry, I didn't explain the issue clearly...
> The assumption is that the height with size=10 should be 10 times the height with size=1 when there is no content or the content is ignored.
Yes, that's true. But there is border for each option. In WebKit, the height of `select` with size=10 is not `10 x "the clientHeight of size=1"`, but `10 x ("the clientHeight of size=1" + border width)`. I personally think WebKit makes more sense, it actually reveals all the options. But this is not the test cares about, so I use a reference instead of the calculation.

Reply:
Ah, I see... Yes, true, it makes sense. Will fix this.